archive/import.go

```diff
@@ -208,12 +220,19 @@ func (dw *tsDirWriter) flush() error {
 		}
 		expr.SortStable(dw.records, importCompareFn(dw.ark))
 		r = zbuf.Array(dw.records).NewReader()
-		defer func() {
-			dw.records = dw.records[:0]
-			dw.addBufSize(dw.bufSize * -1)
-		}()
 	}
-	w, err := newChunkWriter(dw.ctx, dw.ark, dw.uri)
+	first, last := dw.minTs, dw.maxTs
```

Comment:
I know this is kind of late to be making this comment, but: Why not just make this min and max and not worry about having to switch this up based off of sorted direction?

Reply:
You're right, we could do that, and then later in the walk code we could look at the archive's data sort direction to know how to construct the span for the chunk. I like doing it here, though, as it makes the file name reflect the actual order of the records inside.